### ISSUE TITLE
jitsi meet sometimes breaks after switch

### ISSUE THREAD

**Issue by ghost** [NixOS/nixpkgs#117212]

**Comment by dotlambda** [comment]
Does this manifest as "something went wrong" once two people connect?

**Comment by petabyteboy** [comment]
> Does this manifest as "something went wrong" once two people connect?

Yes. `systemctl restart prosody 'ji*'` usually fixes it until the next reboot / switch

**Comment by ajs124** [comment]
Maybe adding some `RestartIfChanged`s somewhere would help?

**Comment by dotlambda** [comment]
cc @tilpner

**Comment by ThePurox** [comment]
Adding `systemd.services.prosody.reloadIfChanged = true;` to the configuration fixed the problem for me.
So I added a PR to fix this for everyone.

**Comment by Al2Klimov** [comment]
```nix
systemd.services.jitsi-videobridge2.after = [ "network-online.target" "prosody.service" ];
```
